sdk/include/psdk/winbase.h

```diff
@@ -210,6 +210,38 @@ extern "C" {
 #define CREATE_SECURE_PROCESS             0x00400000
 #endif // NTDDI_WIN10_RS4
 
+#if (_WIN32_WINNT >= 0x0600)
+#define ProcThreadAttributeValue(Number, Thread, Input, Additive) \
```

Comment:
Note: it would make more sense to have this macro + the `PROC_THREAD_ATTRIBUTE_*` defines down below, moved after the `PROC_THREAD_ATTRIBUTE_NUM` enum typedef instead.